### PR TITLE
Road To Camoflage Color

### PR DESCRIPTION
**Pull request by consciousnessdev**
Adding color library, next is adding camoflage color to framework/modules/_colors.scss

### CODE REVIEW

File: framework/style.css.map [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (adeyahya) [inline framework/style.css.map]: did you copy this code from somewhare else?

Comment (consciousnessdev) [reply framework/style.css.map]: nope, i'd typed it one by one, references of color from w3c, why?

Comment (adeyahya) [reply framework/style.css.map]: no, I asked about `style.css.map` nevermind btw lgtm.

Comment (consciousnessdev) [reply framework/style.css.map]: sass --watch generate it,  because learning webpack still on the fly :smile:

**Comment by consciousnessdev** [comment]
Thanks :pray:

**Pull request merged**